structure/0039-pr-policy.md

```diff
@@ -0,0 +1,74 @@
+A Pull Request is considered to have been approved when all of the following are true:
+- The Pull Request has received the needed amount of Required Approvals and Minimum Team Reviews
+- The Pull Request has no changes requested reviews by any members of the team
+- Nobody in the team has had their review requested on the Pull Request using the system on Github
+    - This includes someone requesting the review of themselves or another person
+
+## Final Comment Period (FCP)
+- Begins immediately after the Acceptance Criteria are met.
+- Lasts a certain amount of days, depending on the category the PR was assigned to.
+- May be ended at any time by a team member if they have legitimate and substantiated concerns.
+
+## Other information
+Team members may dismiss their own reviews.
+
+Admins and technical leads may dismiss outstanding reviews at their discretion, but should only do so if the team member cannot be contacted within a reasonable time-frame. Reviews should *not* be dismissed simply because the majority (or that specific admin/technical lead) disagrees with one person or feels their concerns are invalid. Instead, a formal vote to merge a PR should be used.
+
+## Template for CONTRIBUTING.MD
+### Pull Request Process
+To get a pull request merged into PROJECT, it must get a certain number of approvals from the maintainers, and then it will enter a Final Comment Period. If the pull request passes the final comment period without opposition, the PR will be merged. Otherwise, the PR will return to being in review. 
+
+The exact number of reviews needed, and the length of the Final Comment Period, varies depending on the scope and complexity of the pull request. The numbers for each category are listed below.
+
+#### CATEGORY ONE
+**Required Approvals**: X
+- At least Y of these must come directly from the TEAM team and not its superiors.
+
+**Final Comment Period**: X days
+
+\<Continue for each category>
+
+This is only a summary of the process. The exact rules are defined in [RFC 39](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-policy.md)
+## Conflicts
+If the team cannot agree on whether to merge a PR or deny it, there are steps they can take to avoid decision paralysis. This should be a rare process--teams are encouraged to resolve these disagreements through talking the details out, but sometimes members of the team may have fundamental disagreements over the details of a PR.
+
+In the event the disagreement is over API structure, the PR may, at the team's option, be closed and an RFC be created that defines the feature.
+
+In the event the disagreement is over implementation details, or would otherwise not benefit from being forced through the long RFC process, the voting method defined in RFC 6 can be used. Once a vote is requested, the PR will immediately enter a Final Comment Period that cannot be ended. At the end of this period, the team will vote on if the PR should be merged or closed.  
```

Comment:
> Also, who can request such a vote to be held

This is a good question to ask. Any team member responsible for the PR may request that vote. I'll clarify it in the RFC.

>  Also, who can request such a vote to be held? Is there a minimum period of time after PR creation where such a vote cannot be requested (to prevent those which can request a vote from pushing it forward without due consideration, in the hopes that it will be rejected by the vote)

No, and I don't think that this RFC should focus on trying to avoid people acting in bad faith. If someone is abusing this process to push an agenda, they should be removed from the team.
However, I do see that it would be useful to be able to vote that the PR still needs time to mature. I'll add a clause about that.

Reply:
I though about this a bit further. What I decided on doing is making the voting FCP last two weeks (to give everyone plenty of time to talk out disagreements, make changes to the PR, etc), and adding an option to vote to keep the PR open.